tensorboard/plugins/hparams/hparams_demo.py

```diff
@@ -261,13 +244,7 @@ def run_all(logdir, verbose=False):
   num_sessions = flags.FLAGS.num_session_groups * sessions_per_group
   session_index = 0  # across all session groups
   for group_index in xrange(flags.FLAGS.num_session_groups):
-    hparams = {
-        "conv_layers": rng.randint(1, 3),
-        "conv_kernel_size": rng.choice([3, 5]),
-        "dense_layers": rng.randint(1, 3),
-        "dropout": rng.uniform(0.1, 0.4),
-        "optimizer": rng.choice(["adam", "adagrad"])
-    }
+    hparams = {h: sample_uniform(h.domain, rng) for h in HPARAMS}
```

Comment:
Shouldn't it be `{h.name: sample_uniform(h.domain, rng) for h in HPARAMS}`?

 Perhaps the doc on `run` should be updated. Currently, it reads `hparams: A dict mapping hyperparameter names to values.`.